### PR TITLE
[ansible-test] add freebsd/13.0 remote

### PR DESCRIPTION
**Pull request by relrod**
##### ISSUE TYPE
Change:
- FreeBSD 13.0 was released on the 13th, add it to ansible-test.

Test Plan:
- ci_complete

Signed-off-by: Rick Elrod <rick@elrod.me>

##### SUMMARY

- Test Pull Request

##### COMPONENT NAME

ansible-test

### CODE REVIEW

File: test/integration/targets/setup_paramiko/install-FreeBSD-13-python-2.yml
Comment (samdoran) [inline test/integration/targets/setup_paramiko/install-FreeBSD-13-python-2.yml:6]: Do we want the test fixes in a separate PR? That's helpful if we want to backport these test fixes separate from the addition of FreeBSD 13.

File: .azure-pipelines/azure-pipelines.yml
Comment (samdoran) [inline .azure-pipelines/azure-pipelines.yml:180]: Do we want to remove FreeBSD 11 in `devel`? It was [end of life on September 30, 2020](https://www.freebsd.org/security/unsupported/).

Comment (samdoran) [reply .azure-pipelines/azure-pipelines.yml:180]: ☝️ @mattclay

Comment (relrod) [reply .azure-pipelines/azure-pipelines.yml:180]: I think 11.4 is still supported, it's listed on the main page: https://www.freebsd.org
And [this](https://www.freebsd.org/security/#sup) says it's supported until September of this year.

File: test/integration/targets/setup_paramiko/install-FreeBSD-13-python-2.yml
Comment (relrod) [reply test/integration/targets/setup_paramiko/install-FreeBSD-13-python-2.yml:6]: These changes are only useful if the addition of FreeBSD 13 is backported anyway. This setup role basically just has a bunch of distro-and-version-specific file includes, so when we bump FreeBSD, we need to update it. It's been that way the [last](https://github.com/ansible/ansible/commit/4357a78130d84785720750d3d6801db5cf4df464) few [times](https://github.com/ansible/ansible/commit/11b7091c84ed6cf9576f319118b88f2a81894764) I've updated freebsd as well.

File: .azure-pipelines/azure-pipelines.yml
Comment (samdoran) [reply .azure-pipelines/azure-pipelines.yml:180]: Ah, gotcha. So I suppose we just need to decide if we want to expand the matrix or trim 11.4.

Comment (mattclay) [reply .azure-pipelines/azure-pipelines.yml:180]: Since FreeBSD 11.x is expected to be EOL before Ansible 2.12 is released, we should remove it from CI now and replace it with FreeBSD 13.0.

Once this is merged we should notify collection maintainers so they can make the same changes to their CI configurations. Then as we get closer to releasing Ansible 2.12 (or when collections have all been updated) we can remove support for FreeBSD 11.x from ansible-test.

**Review by mattclay — APPROVED** [review]

**Pull request merged**
